Update README.md (#3)
Dun-sin/HearItFresh: README.md

<div align="center">
  
![image](https://user-images.githubusercontent.com/78784850/210045371-8f386335-88cf-4a65-9d00-6ac0e808269e.png)
  
</div>

<h1 align="center">HearItFresh</h1>

> HearItFresh is a web application that allows users to discover and share new music. Users can sign up for an account, search for and listen to songs, create and share playlists, and follow other users to see their music recommendations.

Reviewer: Once again, try to actually be genuine, rather than using ChatGPT to do this

## 💫 Features

- User authentication and authorization
- Search for songs by title, artist, or genre
- Listen to songs directly in the browser
- Create and share playlists with other users

> - Follow other users to see their music recommendations

Reviewer: This isn't what my app does, did you use chatGPT to generate this readme lol

## 🛠️ Installation

To install and run HearItFresh locally, follow these steps:

1. Clone this repository to your local machine.
2. Navigate to the project directory in your terminal.
3. Run `npm install` to install the project dependencies.

> 4. Set up the database by running `npm run db:setup`.

Reviewer: there's no database

5. Start the server by running `npm start`.
6. Navigate to `http://localhost:3000` in your web browser to use the application.

> Note: You will need to have Node.js and MongoDB installed on your machine to run this application.

Reviewer: there's no backend either

## 👨‍💻 Usage

> To use HearItFresh, simply navigate to the website and sign up for an account. Once logged in, you can search for songs, listen to them, and add them to playlists. You can also follow other users and see their music recommendations on your home page.

Reviewer: This isn't what my project does

<div align="center">

![image](https://user-images.githubusercontent.com/78784850/210045371-8f386335-88cf-4a65-9d00-6ac0e808269e.png)

</div>

<h1 align="center">HearItFresh</h1>

> <h2 align="center">Explore fresh music inspired by the Spotify playlists you ❤️</h2> 

Reviewer: ```suggestion
<h2 align="center">Discover new tracks with Hear It Fresh! This user-friendly web app generates personalized Spotify playlists based on your favourite artists or a Spotify playlist link. Explore fresh music.</h2>
```

## 💫 Features

- Authentication of users is carried out through Spotify Login.

> - Create playlists using artist, track, or playlist seeds as a basis.

Reviewer: ```suggestion
- Create new playlists using your favourite artists or an existing playlist
```

> - Use a personal playlist as a basis to generate new playlists.

Reviewer: ```suggestion
```

> - Tune parameters like different genre, none popular artists etc. to further customize the playlist

Reviewer: ```suggestion
- Get better results by changing the parameters to either getting a playlist of different artists opposite to your taste or get less popular artists
```
